.mocharc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://raw.githubusercontent.com/SchemaStore/schemastore/master/src/schemas/json/mocharc.json",
+  "require": [
+    "source-map-support/register",
+    "test/tools/chai-addons.js"
```

Comment:
copied directly out of mongodb-client-encryption, with minor modifications to remove TS plugin support because we don't test TS.